src/Classes/ServiceAPI/MyRadio_Timeslot.php

```diff
@@ -348,6 +348,17 @@ public static function getMostListened($date = 0)
         return $top;
     }
 
+    public static function getFromSlug($slug)
```

Comment:
I kinda feel like there should be a more general purpose search-by-meta function. If there isn't one already. And if there isn't one already, how did we get this far? :P